svthalia/concrexit
# Simplify thabloids

Closes #2953.
Closes #2018.



### Summary
This removes the thabloid fancybox preview that wasn't working any more. 
Because of that, we no longer need to store jpegs of all pages, and can instead just store a cover image in an ImageField.

### How to test
Steps to test the changes you made:
1. Run the migration and check that covers have been created
2. Upload a new thabloid wiht manually made cover
